Copilot/README.md

```diff
@@ -35,22 +35,53 @@ Watch this [demo](https://www.youtube.com/watch?v=30i7q09EtQo) to learn more abo
 
 ## Getting Started
 
-### 1. Clone this repository and copy SDK files into your React repo
+### 1. Clone this repository and install the SDK into your React repo
 
-```
+```bash
 git clone -b feature/copilot https://github.com/microsoft/SharePoint-Embedded-Samples
+
+cd SharePoint-Embedded-Samples/Samples/spa-azurefunction
+
+npm install "https://download.microsoft.com/download/1315a30d-fe00-45b3-a149-d3235201f8ce/microsoft-sharepointembedded-copilotchat-react-1.0.2.tgz"
```

Comment:
Can this just be added to package.json and including the checksum? So then doing npm install would install everything including the copilot package?